themes/reactos/static/css/style.blue.css

```diff
@@ -25,6 +25,25 @@
 	display: block !important;
 }
 
+/*
+ * This affects mobile devices such as smartphones. The margin of the bottom page is quite higher because
+ * of the footer having an absolute position and that it occupies a large part of the bottom's margin.
+ */
+@media (hover: none) and (pointer: coarse) {
+	body {
+		margin-bottom: 130px;
```

Comment:
This only works if display is wide enough - else this is not enough or becomes too much on some scenarios.